Mathlib/Probability/Moments.lean

```diff
@@ -336,4 +344,27 @@ theorem measure_le_le_exp_cgf [IsFiniteMeasure μ] (ε : ℝ) (ht : t ≤ 0)
 
 end MomentGeneratingFunction
 
+theorem integrable_bounded [IsFiniteMeasure μ] (a b : ℝ) {X : Ω → ℝ} (hX : AEMeasurable X μ)
+    (h : ∀ᵐ ω ∂μ, X ω ∈ Set.Icc a b) :
+     Integrable X μ := by
+   have m1 : HasFiniteIntegral X μ := by
+      apply (hasFiniteIntegral_const (max ‖a‖ ‖b‖)).mono'
+      filter_upwards [h.mono fun ω h ↦ h.1, h.mono fun ω h ↦ h.2] with ω using abs_le_max_abs_abs
+   exact ⟨aestronglyMeasurable_iff_aemeasurable.mpr hX, m1⟩
+
+lemma aemeasurable_expt {X : Ω → ℝ} (t : ℝ) (hX : AEMeasurable X μ) :
+    AEStronglyMeasurable (fun ω ↦ rexp (t * (X ω))) μ :=
+  aestronglyMeasurable_iff_aemeasurable.mpr <| measurable_exp.comp_aemeasurable' (hX.const_mul t)
+
+lemma integrable_expt [IsFiniteMeasure μ] {X : Ω → ℝ} (t b : ℝ) (ht : t > 0)
+    (hX : AEMeasurable X μ) (hb : ∀ᵐ ω ∂μ, X ω ≤ b) :
+    Integrable (fun ω ↦ exp (t * (X ω))) μ := by
```

Comment:
4 comments:
- a suggestion for a better name
- this is also true for `t = 0` so you should use `ht : 0 ≤ t`
- we don't use hypotheses of the shape `t > 0`, but use `0 < t` instead. 
- Some parentheses were not necessary
```suggestion
lemma integrable_exp_mul_of_le [IsFiniteMeasure μ] {X : Ω → ℝ} (t b : ℝ) (ht : 0 ≤ t)
    (hX : AEMeasurable X μ) (hb : ∀ᵐ ω ∂μ, X ω ≤ b) :
    Integrable (fun ω ↦ exp (t * X ω)) μ := by
```